test/modules/packages/Crypto/saru/blowfish/bfTest2.skipif

```diff
@@ -0,0 +1 @@
+../aes/aesTest1.skipif
```

Comment:
This should be a symlink to `../aes/aesTest1.skipif` instead of a file containing that text. e.g.

`ln -s ../aes/aesTest1.skipif bfTest2.skipif`

Reply:
This is what I had done before putting the text. xD